### ISSUE TITLE
Improve automated thresholding activity (for skimage napari)

### ISSUE THREAD

**Issue by tischi** [NEUBIAS/training-resources#663]
[This activity](https://neubias.github.io/training-resources/auto_threshold/index.html#auto) works immediately with two images, especially in the python implementation this is not well motivated and distracting.

I think we should change this activity to just work with one image.

Then there could be a second activity that works on two images...

Any opinions?

ping @jhennies @k-dominik @maulakhan

**Comment by tischi** [comment]
We could also omit the step of finding the threshold manually entirely to make the activity more concise.

It could be interesting to run the automated thresholding algorithms and inspect whether they relate to any obvious features in the history.

**Comment by tischi** [comment]
I think, as @jhennies did today, comparing different thresholding methods on the same image would be much more the point of this module.

**Comment by tischi** [comment]
Also showing multi_otsu makes a lot of sense, imho. 

Would be nice to find an image where this does something interesting; but maybe already the current example is good, because the single otsu fails.

**Comment by jhennies** [comment]
I agree with most of this. I'm not sure if we should drop the manual thresholding entirely. Considering the concept is already taught in the previous module one could dial it down a bit though. 

For example just ask the audience what they would propose as a threshold and then compare this to what the automated method(s) generate

**Comment by tischi** [comment]
I would change this now soon such that:

1. We only work with one image
2. We explore what we get from different automated thresholding methods

Doing this, the disadvantage is that we don't make the point explicitly anymore that the same _automated_ threshold may work on different images, while the same _manual_ threshold may not work on different images. However, that's something one could maybe just say, and one cannot teach everything. Maybe that's also something more for the "batch processing" module.

@manerotoni OK?
